README.md

```diff
@@ -67,7 +67,8 @@ easy-to-use calculator.
   measurements with their physical units, perform numerical integration
   with [`QuadGK.jl`](https://github.com/JuliaMath/QuadGK.jl), numerical and
   automatic differentiation, and much more.
-* Integration with [`Plots.jl`](https://github.com/JuliaPlots/Plots.jl).
+* Integration with [`Plots.jl`](https://github.com/JuliaPlots/Plots.jl) and
```

Comment:
Perhaps you'd want to mention this in the documentation as well: https://github.com/JuliaPhysics/Measurements.jl/blob/daa1ad282a02d945a1f6c69a8198998aa4b63002/docs/src/index.md#features-list